pipelines/matrix/src/matrix/cli_commands/run.py

```diff
@@ -148,6 +151,9 @@ def _run(config: RunConfig, kedro_session: KedroSessionWithFromCatalog) -> None:
 
         from_catalog = _extract_config(config, session)
 
+        # Then reset the env var back
+        os.environ["RUN_NAME"] = "test-run"
```

Comment:
What's this one doing?

Reply:
Are you setting the environment variable and resetting it? I think alternatively we should be able to update the specific `global` entry directly in the instantiated catalog.